### ISSUE TITLE
Create a "credits" animation for acknowledging authors of libraries we use and inspiration

### ISSUE THREAD

**Issue by NickSeagull** [neohaskell/NeoHaskell#52]
A command called `credits` would output a credits animation like in the movies. It'd be cool if it has a "matrix style" background (like the `cmatrix` cli tool) and then a frame in the middle that makes the categories and the libraries/people scroll to the top. Contributors to the NeoHaskell project should be shown too.

**Comment by LemonjamesD** [comment]
This is an unnecessary feature and probably should not be put on this milestone, this just seems childish.

**Comment by moodmosaic** [comment]
That’d be actually a cool thing to see from the command line.

**Comment by LemonjamesD** [comment]
> That’d be actually a cool thing to see from the command line.

Cool != necessary.

**Comment by moodmosaic** [comment]
Yeah, I think whether necessary or not, this issue just keeps tracks of the idea. That’s all…

**Comment by NickSeagull** [comment]
> This is an unnecessary feature and probably should not be put on this milestone, this just seems childish.

Your comment was great until the comma.

Many applications have a window that shows all the license files of the libraries they use. I believe that it is good to acknowledge the tools that we leverage, so they get recognition.

And yeah, I'm childish, the project is childish, and the website is childish. In fact, the only reason that the website font is not Comic Sans, is that Ubuntu Sans is more readable on mobile.

The world is already too serious to putting a mask an "behave like an adult". We should learn more from children and enjoy life to its fullest 😄

**Comment by LemonjamesD** [comment]
> Your comment was great until the comma.
> 
> Many applications have a window that shows all the license files of the libraries they use. I believe that it is good to acknowledge the tools that we leverage, so they get recognition.
> 
> And yeah, I'm childish, the project is childish, and the website is childish. In fact, the only reason that the website font is not Comic Sans, is that Ubuntu Sans is more readable on mobile.
> 
> The world is already too serious to putting a mask an "behave like an adult". We should learn more from children and enjoy life to its fullest 😄

I don't think any project worth it's salt should have animated credits "like in the movies".

**Comment by NickSeagull** [comment]
You're welcome to look for another project worth it's salt
